services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_organization_spec.rb

```diff
@@ -4,6 +4,10 @@
 
 describe VitallyIntegration::SerializeVitallySalesOrganization do
 
+  before do
+    create(:activity_classification, key: 'evidence')
```

Comment:
Again, I'm pretty sure that `create(:evidence)` is defined in the ActivityClassification factory.